seed/views/portfoliomanager.py

```diff
@@ -57,6 +57,11 @@ def template_list(self, request):
                 {'status': 'error', 'message': pme.message},
                 status=status.HTTP_400_BAD_REQUEST
             )
+        except Exception as e:
```

Comment:
This is just a catch-all during the template list retrieval, for anything not covered.